Source/Core/loadArrayBuffer.js

```diff
@@ -13,10 +13,10 @@ define([
      *
      * @exports loadArrayBuffer
      *
-     * @param {String|Promise.<String>} url The URL of the binary data, or a promise for the URL.
+     * @param {String} url The URL of the binary data.
      * @param {Object} [headers] HTTP headers to send with the requests.
-     * @returns {Promise.<ArrayBuffer>} a promise that will resolve to the requested data when loaded.
-     *
+     * @param {Request} [request] The request object.
```

Comment:
We have this identical `@param {Request} [request] The request object.` documentation everywhere but the Request object itself says `Stores information for making a request. This should not be constructed directly. Instead, call the appropriate load function, like `loadWithXhr`, `loadImage`, etc.`  One of them is out of date.  Documentation as a whole could be improved.